src/Components/Articles.jsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../Utils/api";
+import ArticleCard from "./ArticleCard";
+
+const Articles = () => {
+  const [articles, setArticles] = useState([]);
```

Comment:
I would consider having some loading state here so that slower internet connections know when the api request is still ongoing